src/shared/services/utils.ts

```diff
@@ -246,19 +252,20 @@ export const toHumanReadableBytes = input => {
 }
 
 export const getBrowserName = function() {
-  if (!!window.opera || navigator.userAgent.indexOf(' OPR/') >= 0) {
+  if (!!(window as any).opera || navigator.userAgent.indexOf(' OPR/') >= 0) {
     return 'Opera'
   }
+  // @ts-expect-error ts-migrate(2304) FIXME: Cannot find name 'InstallTrigger'.
   if (typeof InstallTrigger !== 'undefined') {
```

Comment:
This seems like a weird way to check for firefox

Reply:
Yes, this code is a few years old, it could probably be simplified now